src/main/java/uk/gov/hmcts/reform/preapi/tasks/CleanupLiveEvents.java

```diff
@@ -30,8 +42,20 @@ public void run() {
                         .getResourceState().equals(LiveEventResourceState.RUNNING.toString())
                     ).forEach(liveEventDTO -> {
                         log.info("Stopping live event {}", liveEventDTO.getId());
-                        // @todo uncomment this line when https://github.com/hmcts/pre-api/pull/579/ is merged
-                        // mediaService.stopLiveEvent(liveEventDTO.getId());
+
+                        var captureSession = captureSessionService.findByLiveEventId(liveEventDTO.getId());
```

Comment:
Some of this code doesn't capture the functionality of the stop live event endpoint. 
The stop live event endpoint creates a recording if there is a recording found. I think you are missing this code: https://github.com/hmcts/pre-api/blob/master/src/main/java/uk/gov/hmcts/reform/preapi/controllers/MediaServiceController.java#L119-L129  and some of this: https://github.com/hmcts/pre-api/blob/master/src/main/java/uk/gov/hmcts/reform/preapi/services/CaptureSessionService.java#L220-L226 . Also this will not update the capture sessions fields like status and finished at/by

Reply:
Think instead code should loop through running live events and for each:
1. Create random id for recording
2. Call `mediaService.stopLiveEvent` and get the status returned
3. Update the capture session via `captureSessionService.stopLiveEvent()` passing the returned status from the media service